tests/integration/engines/test_int_functions.py

```diff
@@ -1553,11 +1585,16 @@ def test_timestamp_seconds(get_session_and_func):
 def test_window(get_session_and_func, get_func):
     session, window = get_session_and_func("window")
     sum = get_func("sum", session)
+    col = get_func("col", session)
     df = session.createDataFrame([(datetime.datetime(2016, 3, 11, 9, 0, 7), 1)]).toDF("date", "val")
-    w = df.groupBy(window("date", "5 seconds")).agg(sum("val").alias("sum"))
+    w = (
+        df.withColumn("window", window("date", "5 seconds"))
+        .groupBy("window")
+        .agg(sum("val").alias("sum"))
+    )
     result = w.select(
-        w.window.start.cast("string").alias("start"),
-        w.window.end.cast("string").alias("end"),
+        col("window.start").cast("string").alias("start"),
+        col("window.end").cast("string").alias("end"),
```

Comment:
Why was this change made? For context I try to stick as close as possible to the examples from PySpark documentation in order to validate consistent behavior/compatibility: https://spark.apache.org/docs/3.4.0/api/python/reference/pyspark.sql/api/pyspark.sql.functions.window.html

Reply:
Hi,
I've changed it because sqlframe does not support that syntax. Right now this test is being skipped by all the backends except databricks so that´s why it didn't failed before.

Right now the sqlframe Column class does not support the "dot notation" so doing something like `col.start` fails.
![image](https://github.com/user-attachments/assets/f8012ee9-32da-4267-82e3-0fdff6aa5f0d)

Reply:
Hi, 
I've added a comment on all the tests using this syntax